native/ios/native.xcodeproj/project.pbxproj

```diff
@@ -5,6 +5,7 @@
 	};
 	objectVersion = 46;
 	objects = {
+
```

Comment:
How come this file changed? are there new native deps?

Reply:
No clue lol